Sources/ArcGISToolkit/Components/Augmented Reality/CalibrationView.swift

```diff
@@ -0,0 +1,170 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ARKit
+import SwiftUI
+import ArcGIS
+
+extension WorldScaleGeoTrackingSceneView {
+    /// A view that allows the user to calibrate the heading of the scene view camera controller.
+    struct CalibrationView: View {
+        @ObservedObject var viewModel: ViewModel
+        /// The slider value for the camera heading.
+        @State private var headingSliderValue = 0.0
+        /// The slider value for the camera elevation.
+        @State private var elevationSliderValue = 0.0
+        /// The elevation timer for the "joystick" behavior.
+        @State private var headingTimer: Timer?
+        /// The heading timer for the "joystick" behavior.
+        @State private var elevationTimer: Timer?
+        /// The heading delta amount based on the heading slider value.
+        private var joystickHeadingDelta: Double {
+            Double(signOf: headingSliderValue, magnitudeOf: headingSliderValue * headingSliderValue / 25)
+        }
+        /// The elevation delta amount based on the elevation slider value.
+        private var joystickElevationDelta: Double {
+            Double(signOf: elevationSliderValue, magnitudeOf: elevationSliderValue * elevationSliderValue / 100)
+        }
+        
+        var body: some View {
+            Button {
+                viewModel.isCalibrating = true
+                setBasemapOpacity(0.5)
+            } label: {
+                Text("Calibrate")
+            }
+            .popover(isPresented: $viewModel.isCalibrating) {
+                VStack {
+                    Text("Heading: \(viewModel.calibrationHeading?.rounded(.towardZero) ?? viewModel.cameraController.originCamera.heading.rounded(.towardZero), format: .number)")
+                    
+                    HStack {
+                        Button {
+                            let heading = viewModel.cameraController.originCamera.heading - 1
+                            updateHeading(heading)
+                        } label: {
+                            Image(systemName: "minus")
```

Comment:
Maybe consider using `.imageScale(.large)` here. When I tried to tap the button, it is a bit small to me.